### PR TITLE
Fix bug of not respecting dilation argument in RepVGGBlock

### PR DESCRIPTION
**Pull request by BloodAxe**

### CODE REVIEW

**Review by ofrimasad — COMMENTED** [review]
wouldn't this be cleaner if we add 'padding' parameter?

File: src/super_gradients/modules/repvgg_block.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def __init__(
             dilation=dilation,
             kernel_size=3,
             stride=stride,
-            padding=1,
+            padding=dilation,
```
[the remark this answers is not in the document]
Comment (BloodAxe) [reply src/super_gradients/modules/repvgg_block.py:72]: Yes, but there is no controversy. When using dilation we must also set padding respectfully to: ` (kernel_size//2) *  dilation` which for 3x3 conv becomes just `dilation`

[the remark this answers is not in the document]
Comment (BloodAxe) [reply src/super_gradients/modules/repvgg_block.py:72]: @ofrimasad  Padding is a function of kernel size and dilation factor (padding = dilation * kernel_size//2)
* Kernel size is fixed in this block to 3x3 and it cannot be changed (RepVGG reparametrization logic tailored for 3x3 kernel).
* However user can specify `dilation` factor (which we already have in args).

So we don't need user to provide `padding` parameter as we have all information that is necessary to instantiate block correctly.

**Review by ofrimasad — DISMISSED** [review]
LGTM

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**